addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -966,7 +966,9 @@ export class OdooEditor extends EventTarget {
             }
         }
         if (sideEffect) {
-            this._activateContenteditable();
+            if (!this._fixLinkMutatedElements) {
```

Comment:
I'm confused by the commit message. It looks like there is half a sentence remaining between the explanation of the issue and the task number.